stacks/java-openliberty-gradle/devfile.yaml

```diff
@@ -18,9 +18,8 @@
 schemaVersion: 2.1.0
 metadata:
   name: java-openliberty-gradle
-  version: 0.4.0
-  displayName: 'Open Liberty Gradle'
-  description: Java application Gradle-built stack using the Open Liberty runtime
+  displayName: Open Liberty Gradle
+  description: Java application based on OpenJ 9 Java 11, Gradle using the Open Liberty runtime 22.0.0.1
```

Comment:
This doesn't make sense :-/  Is OpenJ supposed to be OpenJDK? Then how can it be OpenJDK 9 and Java 11 at the same time?